core/model/__init__.py

```diff
@@ -515,11 +515,7 @@ def _bulk_operation(self):
     SAMLFederation,
 )
 from core.model.admin import Admin, AdminRole
-from core.model.cachedfeed import (
-    CachedFeed,
-    CachedMARCFile,
-    WillNotGenerateExpensiveFeed,
-)
+from core.model.cachedfeed import CachedMARCFile, WillNotGenerateExpensiveFeed
```

Comment:
When we remove `WillNotGenerateExpensiveFeed`, it should also be removed from this import.